modules/tinymce/src/themes/silver/main/ts/ContextToolbar.ts

```diff
@@ -174,41 +176,48 @@ const register = (editor: Editor, registryContextToolbars, sink, extras) => {
 
   const getScopes: () => ScopedToolbars = Thunk.cached(() => {
     return ToolbarScopes.categorise(registryContextToolbars, (toolbarApi) => {
-      const alloySpec = buildToolbar(toolbarApi);
+      // TODO should this get multiple ctx too?
+      const alloySpec = buildToolbar([toolbarApi]);
```

Comment:
This opens the context form, which we said should only have one instance open anyways IIRC, so this should be fine?

Reply:
Yep. Was trying to figure out if it can be triggered in other ways in the future, and if I should at least leave a comment. Might do so - add an assumptions comment or something.